packages/client/modules/meeting/components/ReflectTemplateListTeam.tsx

```diff
@@ -53,9 +70,12 @@ const ReflectTemplateListTeam = (props: Props) => {
       </Message>
     )
   }
+  if (filteredTemplates.length === 0) {
+    return <Message>{`No team templates match your search query "${templateSearchQuery}"`}</Message>
+  }
   return (
     <TemplateList>
-      {teamTemplates.map((template) => {
+      {filteredTemplates.map((template) => {
```

Comment:
+1 it's nice to see the search query highlighted or made bold in the result. something like this: https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/client/components/TypeAheadLabel.tsx

Reply:
Yeah, I do like seeing the text become bold when it matches the query. As the title is already bold here, I've tried highlighting it instead. I kinda like it, not sure if it feels too busy. What do you think?

![template-search](https://user-images.githubusercontent.com/39854876/175565410-79196244-ed15-4940-a3b2-19563de5d6f8.gif)

Reply:
i like it! I was gonna suggest going a shade lighter, but it passes the contrast ratio test, so I'm happy!
<img width="333" alt="Screen Shot 2022-06-24 at 11 50 51 AM" src="https://user-images.githubusercontent.com/5514175/175647250-eda12891-2205-4ac4-98d4-6db42208c047.png">